<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Cost">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Hardness">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий заказ:</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>По возрастанию стоимости</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По уменьшению стоимости</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По возрастанию сложности</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По уменьшению сложности</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Заказы с максимальными сроками</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Заказы с минимальными сроками</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Показывать элементов</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать заказ</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>411</width>
       <height>381</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Архив</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать
существующий
заказ</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>101</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Действия</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать
существующий
заказ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Архив</string>
     </property>
    </widget>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>widget</zorder>
   </widget>
   <widget class="QTableWidget" name="Layout">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>311</width>
      <height>371</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Человекочасы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Расходы на материалы</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>231</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateZAKAZ">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать заказ</string>
    </property>
   </widget>
   <zorder>Cost</zorder>
   <zorder>Hardness</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>Layout</zorder>
   <zorder>listWidget</zorder>
   <zorder>CreateZAKAZ</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>itemActivated(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>Galocka(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>AddElement(element)</signal>
  <slot>Galocka(QListWidgetItem*)</slot>
  <slot>Table(QTableWidget)</slot>
 </slots>
</ui>
